Makefile

```diff
@@ -90,7 +90,7 @@ LDFLAGS += -X "github.com/mattermost/mattermost-server/v6/model.BuildEnterpriseR
 GO_MAJOR_VERSION = $(shell $(GO) version | cut -c 14- | cut -d' ' -f1 | cut -d'.' -f1)
 GO_MINOR_VERSION = $(shell $(GO) version | cut -c 14- | cut -d' ' -f1 | cut -d'.' -f2)
 MINIMUM_SUPPORTED_GO_MAJOR_VERSION = 1
-MINIMUM_SUPPORTED_GO_MINOR_VERSION = 15
+MINIMUM_SUPPORTED_GO_MINOR_VERSION = 16
```

Comment:
Thank you @thierrymarianne for looking into this!

The current suggestion indeed works great. But our release team wants to deduce which Go version to build with from Makefile variables. Which means that we need to capture the patch release version as well. And if we do that, we anyways need to update the value every time we bump it. So I don't think there's a big benefit of getting the value from go.mod. Because go.mod does not capture the patch release version, it's difficult to do this automatically.

Reply:
Hey @thierrymarianne - we discussed internally. We will be taking another approach for the release team. Your current approach looks fine. Thanks!